ssl.go

```diff
@@ -16,9 +15,9 @@ type ZoneCustomSSL struct {
 	Status        string     `json:"status"`
 	BundleMethod  string     `json:"bundle_method"`
 	ZoneID        string     `json:"zone_id"`
-	UploadedOn    time.Time  `json:"uploaded_on"`
```

Comment:
We would like to keep dates as `time.Time` types instead of strings, as they are easier to work with in Go. Is there any specific reason for this change?